README.md

```diff
@@ -79,6 +80,39 @@ then add this to your maven configuration:
 </dependencies>
 ```
 
+### Gradle
+For Gradle there are 2 plugins that help you keep from unnecessarily compiling the `dagger-compiler` library into your project. One for POJO projects and another for Android projects. They allow you to add the `dagger-compiler` artifact as an `apt` dependency.
```

Comment:
I would split this into two blocks. One for Java Gradle and one for Android Gradle. You'll only confuse people by mixing them.